src/main.rs

```diff
@@ -67,7 +69,13 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    let mut days = 0;
+    let mut rabbits: i128 = starting_rabbits;
+    while rabbits > 1 {
+        rabbits = if rabbits%2 == 0 { rabbits/2 } else { (3*rabbits) + 1 };
+        days += 1;
+    }
+    return days;
```

Comment:
In Rust, when returning from a function, we don't actually need to write the return keyword like `return x;` if it's the last expression. Instead, we can just do `x`. For example:

```rust
fn foo() -> i32 {
    let x = 5;
    x
}
```

Or, to simplify that more:

```rust
fn foo() -> i32 {
    5
}
```

Here are some more docs on that: https://doc.rust-lang.org/rust-by-example/fn.html